doc/api/child_process.markdown

```diff
@@ -176,8 +176,8 @@ replace the existing process and uses a shell to execute the command.*
 
 ### child_process.execFile(file[, args][, options][, callback])
 
-* `file` {String} The filename of the program to run
-* `args` {Array} List of string arguments
+* `file` {String} An absolute path to an executable file.
```

Comment:
The "if there is a slash it won't be looked up in the PATH" is an interesting gotcha, worth mentioning... but would require confirmation that is how it works on windows, too, so the docs can be correct for both systems.